Lib/test/test_clinic.py

```diff
@@ -1408,6 +1480,26 @@ def test_scaffolding(self):
         actual = stdout.getvalue()
         self.assertEqual(actual, expected)
 
+    def test_non_ascii_character_in_docstring(self):
+        block = """
+            module test
+            test.fn
+            docstring fü bár baß
+        """
+        with support.captured_stdout() as stdout:
+            self.parse(block)
+        msg = "Non-ascii character appear in docstring"
```

Comment:
The error message here doesn't make grammatical sense!

https://github.com/python/cpython/blob/dfb55d9d7f9349582a2077f87e95c8f9d5e2ebd4/Tools/clinic/clinic.py#L788

We should fix that rather than asserting incorrect behaviour in a test :)

Reply:
Tomorrow! :)

Reply:
How about _"Non-ascii characters are not allowed in docstrings: {the offending char!r}"_

Reply:
> How about _"Non-ascii characters are not allowed in docstrings: {the offending char!r}"_

Sounds good!

Reply:
Addressed in gh-107559 (issue) and gh-107560 (PR).